.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -114,7 +114,7 @@ body:
     id: architecture
     attributes:
       label: Architecture
-      description: What React Native architecture your application is running on?
+      description: What React Native architecture your application is running on? Currently, the default architecture on React Native is Paper so if you haven't changed it your application select this option.
```

Comment:
is this comment necessary?

Reply:
My reasoning is: 
When users enable Fabric they are well aware there are 2 architectures. Users that never heard of it before don't know there are many architectures.

Reply:
ok lol